dynawo/sources/Solvers/FixedTimeStep/SolverTRAP/test/TestBasics.cpp

```diff
@@ -341,665 +293,5 @@ TEST(SimulationTest, testSolverTRAPTestBeta) {
   ASSERT_DOUBLE_EQUALS_DYNAWO(yp[0], 1);
   ASSERT_DOUBLE_EQUALS_DYNAWO(z[0], 1);
 }
-/*
-TEST(SimulationTest, testSolverTRAPTestBetaUnstableRoot) {
```

Comment:
How come those tests were commented?

Reply:
This test file was a copy paste of the one for SIM and only one of the multiple tests was added, and we did not see the rest was commented out. I am not even sure the testBeta is completly relevant as all those tests were designed for the SIM solver. We would need to have a closer look later